src/frontend/screens/Library/components/GameCard/index.css

```diff
@@ -13,12 +13,16 @@
   aspect-ratio: 173/275;
 }
 
+.gameCard.gamepad {
+  aspect-ratio: 3/4;
+}
+
 .gameCard.justPlayed {
   aspect-ratio: 275/205;
 }
 
-.gameCard.gamepad {
-  aspect-ratio: 3/4;
+.gameCard.gamepad.justPlayed {
+  aspect-ratio: 328/205;
```

Comment:
we need a different ratio for the gamepad layout cause we remove the part with the play/settings/update buttons